Task Liverpool/src/test/java/com/encora/automation/test/HomeTest.java

```diff
@@ -1,12 +1,33 @@
 package com.encora.automation.test;
 
+import org.testng.Assert;
+import pages.HomePage;
+import pages.ProductPage;
 import org.testng.annotations.Test;
 
 public class HomeTest extends BaseTest{
 
+    private HomePage homePage;
+    private ProductPage productPage;
+
+    private HomeTest(){
+        homePage = new HomePage(getDriver());
+        productPage = new ProductPage(getDriver());
+    }
+
     @Test
     private void TC1_SearchWithCode (){
```

Comment:
Method name in java should be camelCase

Reply:
Also it's a good practice to add a description to the test

Reply:
fixed.